.drone.yml

```diff
@@ -2,10 +2,18 @@ matrix:
   SBT_RUN:
     - sbt "+bridgeIntegration/cachedPublishLocal" \
           "benchmarks/compile" \
+          "git status" \
           "setupTests" \
+          "git status" \
```

Comment:
Can't we have something that fails if the repository is not clean? That much clearer and easier to debug than having to skim through thousands of lines of logs.

Reply:
I propose we have this here momentarily and then, in another PR, we override the way Drone clones the repository. There's an option in `git submodule` called `--checkout` that should allow us to force the commit hash that we recorded last time, instead of getting the most recent upstream.
  
All of our problems seem to be because we update the branches and our previous PRs haven't registered that commit as the HEAD. Doing that would solve all our problems.

Reply:
Not all our problems. Setting up the scripted tests sometimes makes the repositories dirty.

Reply:
That's true, forgot about it. But how can we have something that fails if the repository is not clean *while running `scripted`*? The only way I see is to hijack scripted, and it doesn't sound exciting...
  
This problem should not be an issue now that we have stable HEADs in all the projects. But I agree it's frustrating when adding new projects or updating them. I don't think there's a good solution though.

Reply:
I meant having something like this: https://github.com/scalacenter/bloop/pull/124/commits/14c2fa24834c17cad3d8629c4f76f01e1e5048e6

It doesn't matter, we'll change that if it bites us again.

Reply:
But `checkClean` runs before `setupTests`, right? In that case, that will never be able to identify which changes happened after `setupTests` fails. So this use case would be fixed in the future PR in which we force `--checkout`, since by definition it will always be clean then :smile:.

Reply:
Yeah, but it runs after `scriptedAddSbtBloop` too 😉